plugin-tester/src/index.js

```diff
@@ -1,12 +1,18 @@
 #! /usr/bin/env node
 
+const serviceAccount = require('./firebase-secrets.json');
```

Comment:
this shouldn't be forced, we should only apply these a user has specified a `firebase-secrets` via flag